lms/envs/common.py

```diff
@@ -2088,3 +2088,19 @@
         'profile_image',
     ],
 }
+
+# PROFILE IMAGE CONFIG
+# TODO: add these settings to aws.py as well
+PROFILE_IMAGE_BACKEND = 'django.core.files.storage.FileSystemStorage'
+# PROFILE_IMAGE_DOMAIN points to the domain from which we serve image
+# files from.  When this is '/', it refers to the same domain as the
+# app server.  If serving from a different domain, specify that here
+# i.e. 'http://www.example-image-server.com/'
+PROFILE_IMAGE_DOMAIN = '/'
+PROFILE_IMAGE_URL_PATH = 'media/profile_images/'
+PROFILE_IMAGE_DEFAULT_FILENAME = 'default_profile_image'  # TODO: determine final name
```

Comment:
@e0d @maxrothman FYI for configurations of ephemeral environments (devstack/fullstack, sandboxes) would be best to copy the placeholder images into place as part of provisioning.  e.g. in the filesystem-backed scenario we'd move some assets from a place in the code repo over to `/edx/var/edxapp/{PROFILE_IMAGE_URL_PATH}`.  I assume we need to file a ticket for ansible changes to do this.  Please advise if we should be specifying settings any differently than they are here in order to make that happen.